### PR TITLE
LPS-64753

### PR DESCRIPTION
**Pull request by Ayoub3r**

### CODE REVIEW

**Comment by pedroamarall** [comment]
ci:test:sf

**Comment by Ayoub3r** [comment]
Hey @pedroamarall, I just corrected the formatting issues I had, could you please re-run the "ci:test:sf" command please?

File: portal-web/test/functional/com/liferay/portalweb/macros/PoshiExercises.macro [hunk 1]
```diff
@@ -0,0 +1,242 @@
+definition {
+
+	macro AddBlogEntry {
+		JSONBlog.addEntry(
+			entryContent = ${entryContent},
+			entryTitle = ${entryTitle});
+	}
```
Comment (manoelcyreno) [inline portal-web/test/functional/com/liferay/portalweb/macros/PoshiExercises.macro:7]: you dont need to create a macro to call only 1 another macro ...

File: portal-web/test/functional/com/liferay/portalweb/macros/PoshiExercises.macro [hunk 2]
```diff
@@ -0,0 +1,242 @@
...
+			fieldValue = ${fieldValue_AgeBr});
+
+		Form.save();
+	}
+
+	macro CheckForStringConfiguration {
+		AssertTextPresent(
+			locator1 = "ApplicationsMenu#CATEGORY",
+			value1 = ${stringValue});
+	}
```
Comment (manoelcyreno) [inline portal-web/test/functional/com/liferay/portalweb/macros/PoshiExercises.macro:78]: you dont need to create a macro to call only 1 another macro ...

File: portal-web/test/functional/com/liferay/portalweb/macros/PoshiExercises.macro [hunk 3]
```diff
@@ -0,0 +1,242 @@
...
+		Form.save();
+	}
+
+	macro editBlogContent {
+		JSONBlog.updateEntry(
+			entryContent = ${entryContent},
+			entryTitle = ${entryTitle},
+			groupName = ${groupName},
+			newEntryContent = ${newEntryContent});
+	}
```
Comment (manoelcyreno) [inline portal-web/test/functional/com/liferay/portalweb/macros/PoshiExercises.macro:126]: you dont need to create a macro to call only 1 another macro ...

File: portal-web/test/functional/com/liferay/portalweb/macros/PoshiExercises.macro [hunk 4]
```diff
@@ -0,0 +1,242 @@
...
+		FormViewBuilder.validateFieldLabel(fieldLabel = ${fieldLabel_Age});
+
+		FormFields.editText(
+			fieldName = ${fieldName_Age},
+			fieldValue = ${fieldValue_Age});
+
+		FormPortlet.clickSubmit();
+
+		Form.submitAgain();
+	}
```
Comment (manoelcyreno) [inline portal-web/test/functional/com/liferay/portalweb/macros/PoshiExercises.macro:240]: Here that validates, not looks that validate, looks that you are doing something and after that validating ...
and are you not using any assert to check the validate part?

**Comment by pedroamarall** [comment]
ci:test:sf

**Review by manoelcyreno — CHANGES_REQUESTED** [review]

**Pull request closed without merging**